docs/alerts/runbooks/resource-reservation-exceeds-quota.md

```diff
@@ -0,0 +1,56 @@
+# Resource Reservation Exceeds Quota
+
+## Severity: Info
+
+## Impact
+
+Knowledge of over requested resources will allow the user to adjust the nominal quota or resources requested by a workload.
+
+## Summary
+
+This alert is triggered when resource reservation is 10 times the available nominal quota in a cluster queue.
+
+## Steps
+
+1. Check current resource reservation for the cluster queue and ensure that the nominal quota for the resource in question is correctly configured. Update the cluster-queue-name in the script below to describe the cluster queue.
+```bash
+cluster_queue=< cluster-queue-name >
+oc describe clusterqueue $cluster_queue
+```
+
+If you would just like to view the Flavors Reservation and Flavors Usage you can use the following command:
```

Comment:
Just a small nit question, wondering if points like this should be bulleted to make them in line with the numbered steps?

Reply:
I think in this case a bullet point makes sense. Updated, thanks!